### PR TITLE
Sublime dev

### PR DESCRIPTION
**Pull request by MasseGuillaume**
Resurecting: https://github.com/NixOS/nixpkgs/pull/26718

### CODE REVIEW

**Comment by jtojnar** [comment]
Quoting the original PR:

> It looks like it should be possible to share most of the file with stable version. Could it be factored out to `common.nix`?

**Comment by MasseGuillaume** [comment]
@jtojnar done.

**Comment by jtojnar** [comment]
Great, thanks. Although I still do not like the argument duplication very much. Could you do something like, for example, [Firefox](https://github.com/NixOS/nixpkgs/blob/729cd0352de9e68936794b8794ef4e56a31cd450/pkgs/applications/networking/browsers/firefox/packages.nix#L5) does?

**Comment by MasseGuillaume** [comment]
@jtojnar done.

**Review by jtojnar — APPROVED** [review]
LGTM, cc maintainers @wmertens, @demin-dmitriy, @zimbatm

File: pkgs/applications/editors/sublime/3/packages.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      x32sha256 = "0vib7d6g47pxkr41isarig985l2msgal6ad9b9qx497aa8v031r5";
+      x64sha256 = "0dy2w3crb1079w1n3pj37hy4qklvblrl742nrd3n4c7rzmzsg71b";
+    } {};
+
+    sublime3 = common rec {
+      buildVersion = "3143";
+      x32sha256 = "0dgpx4wij2m77f478p746qadavab172166bghxmj7fb61nvw9v5i";
+      x64sha256 = "06b554d2cvpxc976rvh89ix3kqc7klnngvk070xrs8wbyb221qcw";    
+    } {};
+  }
```
Comment (zimbatm) [inline pkgs/applications/editors/sublime/3/packages.nix]: the `rec` keywords in this file don't seem to be necessary. `rec` is useful to re-export keys of the current attrset as similarly-named variables into the local scope.

**Review by zimbatm — APPROVED** [review]
looks good overall, no objections for a merge

**Comment by MasseGuillaume** [comment]
@zimbatm @jtojnar everything is done on my side.

**Comment by jtojnar** [comment]
Right, sorry, fixed in 8e18b7c409600faf9b60685944a00fb1bb4f1c3f

**Pull request merged**